### PR TITLE
fix(workordercell): improve component and fix deviations

### PR DESCRIPTION
**Pull request by Jonas-Omdal**
***Key changes:***

- Replaced moment with Date.
- Removed the `Ready for execution` icon and label. 
- If the status do not include STRT it should show a new label and icon.
- Simplified the component, remove unnecessary props and logic.
       - Renamed the workOrder prop to workOrderId.
       - Add new prop, workOrderType.
       - remove the symbolDirection prop. Symbols should only be displayed in column.
       
 Fixes: #622

### CODE REVIEW

File: .changeset/tricky-deers-argue.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@equinor/mad-dfw": minor
+---
+
+WorkOrderCell adjusted. Renamed prop `workOrder` to `workOrderID`. Added new prop, `workOrderType`,
```
Comment (HaakonSvane) [inline .changeset/tricky-deers-argue.md:5]: "d" in "ID" should be lowercased :)

File: .changeset/tricky-deers-argue.md
Comment (HaakonSvane) [inline .changeset/tricky-deers-argue.md:1]: Didn't you also remove the override options?

File: packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCell.tsx [hunk 2]
```diff
@@ -2,40 +2,43 @@ import {
...
     const breakpoint = useBreakpoint();
-    const styles = useStyles(themeStyles, { symbolDirection });
+    const token = useToken();
+    const styles = useStyles(themeStyles);
 
     const anyButtonVisible =
         startJobButton?.visible ?? readyForOperationButton?.visible ?? tecoButton?.visible;
-    const currentDate = moment();
+
+    const currentDate = useMemo(() => new Date(), []);
```
Comment (HaakonSvane) [inline packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCell.tsx:40]: Just a question of curiosity, not related to your code changes that much.
Why is this current date being passed in as a prop to the `PropertyList` component? Should it not be able to get this date itself?

File: packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCell.tsx [hunk 3]
```diff
@@ -50,23 +53,28 @@ export const WorkOrderCell = ({
...
-                overwriteLabel={overwriteLabel}
-                valueColor={valueColor}
-                currentDate={currentDate.toDate()}
-            />
+            <View style={styles.dataContainer}>
+                <Typography group="paragraph" variant="body_short" color="textTertiary">
+                    {maintenanceType}
+                </Typography>
+                <PropertyRow label={workOrderType} value={workOrderId} />
+                <PropertyList workOrder={rest} valueColor={valueColor} currentDate={currentDate} />
```
Comment (HaakonSvane) [inline packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCell.tsx:76]: the `workOrder` prop on this component seems to a little bit misleading now that we are no longer passing the whole object in. Maybe this prop should be renamed to something more generic to help communicate that it's job is to take in any object that will be rendered in the list (if the key exists in the label mapping)

File: packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCellNavigation.tsx [hunk 4]
```diff
@@ -13,19 +13,20 @@ type WorkOrderCellNavigationProps = WorkOrderCellProps & {
...
     showSymbols = true,
-    overwriteLabel,
     onPress,
     style,
     ...rest
 }: WorkOrderCellNavigationProps) => {
     const styles = useStyles(themeStyles);
 
-    const currentDate = moment();
+    const currentDate = useMemo(() => new Date(), []);
```
Comment (HaakonSvane) [inline packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCellNavigation.tsx:29]: Same here as for my comment on the WorkOrderCell in regards to passing this as a prop into the PropertyList component

**Review by HaakonSvane — APPROVED** [review]
👍 😎 Life is good, code is good 😎 👍

File: packages/dfw/src/dfwcomponents/WorkOrderCell/PropertyList.tsx [hunk 5]
```diff
@@ -18,34 +17,24 @@ const defaultWorkOrderLabelMap: Record<keyof WorkOrder, string> = {
...
-    workOrder,
-    valueColor,
-    currentDate,
-    overwriteLabel = {},
-}: PropertyListProps) => {
-    const token = useToken();
-
-    const formatDate = (dateString: string) => moment(dateString).format("DD.MM.YYYY");
-
+export const PropertyList = ({ data: workOrder, valueColor }: PropertyListProps) => {
```
Comment (HaakonSvane) [inline packages/dfw/src/dfwcomponents/WorkOrderCell/PropertyList.tsx:24]: rename-refactor seems to have still kept the name of this field `workOrder` internally in this component

**Review by HaakonSvane — DISMISSED** [review]
🤙  Cool code dude 🤙

File: packages/dfw/src/dfwcomponents/WorkOrderCell/PropertyList.tsx (hunk 5 above)
Comment (Jonas-Omdal) [reply packages/dfw/src/dfwcomponents/WorkOrderCell/PropertyList.tsx:24]: ops

**Review by HaakonSvane — APPROVED** [review]
👍

**Pull request merged**